app/backend/src/tokenstorage/token-verify.service.ts

```diff
@@ -16,17 +16,29 @@ export default class TokenIsVerified implements CanActivate {
   }
 
   async canActivate(context: ExecutionContext): Promise<boolean> {
```

Comment:
This file is not a service, it's a Guard

Reply:
It is used as a service also